### PR TITLE
Better Getting Started Guide

### PR DESCRIPTION
**Pull request by plaindocs**
Explain that you run your own agents CC @JuanitoFatas 

@scopegate could you help me out with some styling of the list of install links? Let me know if you'd like the whole lot in html, or if I can just wrap a bunch of markdown in single html class.

### CODE REVIEW

**Comment by nikwek** [comment]
❤️ this

**Comment by plaindocs** [comment]
Hey @nikwek that formatting is just a placeholder while I get some design help from @scopegate , it is definitely suboptimal as is.

File: pages/tutorials/getting_started.md.erb [hunk 1]
```diff
@@ -4,29 +4,40 @@ This guide will help you setup your first Buildkite pipeline. If you don’t hav
...
 
 <a class="inline-block" href="https://buildkite.com/new?template=https://github.com/buildkite/bash-example" target="__blank" rel="nofollow"><img src="https://buildkite.com/button.svg" alt="Add Bash Example to Buildkite" class="no-decoration" width="160" height="30"></a>
 
-## Create your first build
+Accept the pre-filled defaults, and click *Create Pipeline*.
+
+## Run your first build
 
-Now that you’ve started an agent and created a pipeline you’re ready to run your first build. Click the *New Build* button in the top right corner:
+Now that you’re running an agent and created a pipeline you’re ready to run your first build. Click the *New Build* button in the top right corner:
```
Comment (plaindocs) [inline pages/tutorials/getting_started.md.erb:32]:
```suggestion
Now that you’re running an agent and have created a pipeline you’re ready to run your first build. Click the *New Build* button in the top right corner:
```

**Comment by plaindocs** [comment]
I've removed the OS list, so we can merge this, will revisit that later.

**Review by JuanitoFatas — APPROVED** [review]
Thanks!

**Pull request merged**